Configs/.config/hypr/keybindings.conf

```diff
@@ -25,7 +25,7 @@ bind = Alt, Return, fullscreen, # toggle the window between focus and fullscreen
 bind = $mainMod, L, exec, swaylock # launch lock screen
 bind = $mainMod+Shift, F, exec, $scrPath/windowpin.sh # toggle pin on focused window
 bind = $mainMod, Backspace, exec, $scrPath/logoutlaunch.sh # launch logout menu
-bind = Ctrl+Alt, W, exec, killall waybar || (wbarconfgen.sh && waybar) # toggle waybar and reload config
+bind = Ctrl+Alt, W, exec, killall waybar || (env reload_flag=1 wbarconfgen.sh) # toggle waybar and reload config
```

Comment:
My bad. Let’s stick with the conventions. 


``` hyprlang
bind = Ctrl+Alt, W, exec, killall waybar || (env reload_flag=1 $scrPath/wbarconfgen.sh) # toggle waybar and reload config
```

Reply:
should be good now